webpack/scenes/Hosts/ChangeContentSource/components/ContentSourceTemplate.js

```diff
@@ -39,25 +40,25 @@ const ContentSourceTemplate = ({ template, jobInvocationPath }) => {
   return (
     <Grid>
       <GridItem span={7}>
-        <h1>
-          {__("What's next?")}
-        </h1>
-        <p>
-          {jobInvocationPath && (
-            <>
-              <a href={jobInvocationPath}>
-                {__('Run job invocation')}
-              </a>
+        <Alert variant="info" title="Hosts successfully updated" className="margin-top-20">
+          <p>
+            {jobInvocationPath && (
+              <>
+                <a href={jobInvocationPath}>
+                  {__('Run job invocation')}
+                </a>
+            &nbsp;
+                {__('to update configuration on the hosts, or')}
+              </>
+            )}
           &nbsp;
-              {__('to update configuration on all hosts, or')}
-            </>
-          )}
-        &nbsp;
-          {__('update configuration on the hosts manually:')}
-        </p>
+            {__('update configuration on the hosts manually:')}
+          </p>
+        </Alert>
```

Comment:
To ensure proper translation, please use `<FormattedMessage>` similar to https://github.com/Katello/katello/blob/47492881d123b13c2e7973c4e1db965971d45657/webpack/scenes/Subscriptions/SubscriptionsPage.js#L223-L235